ui/src/common/storage.js

```diff
@@ -15,7 +31,8 @@ function isStorageAvailable() {
         e.code === 1014 ||
         e.name === 'QuotaExceededError' ||
         e.name === 'NS_ERROR_DOM_QUOTA_REACHED') &&
-      (storage && storage.length !== 0)
+      storage &&
+      storage.length !== 0
```

Comment:
why you removed the parenthesis?

Reply:
I believe it was prettier who did this or it must have been by accident, I'll revert this change